.github/CODEOWNERS

```diff
@@ -1,6 +1,101 @@
 # Per-directory ownership and automatic assignment for pull requests.
+
+# Audio and Translation team.
+/core/templates/dev/head/exploration_editor/translation_tab/ @DubeySandeep
+/core/controllers/translator*.py @DubeySandeep
+
+
+# Collection project.
+/core/domain/collection_*.py @kevinlee12
```

Comment:
Do you need the "_" before the "*"? (See translator*.py above.)

Reply:
Done!